charts/opentelemetry-kube-stack/values.yaml

```diff
@@ -341,3 +341,172 @@ clusterRole:
 
   # Rules for the cluster role
   rules: []
+
+# Instrumentation configuration
+instrumentation:
+  # Whether instrumentation is enabled or not
+  enabled: false
+  labels: {}
+  annotations: {}
+
+  # Exporter configuration
+  exporter:
+    endpoint: http://collector:55678
```

Comment:
This should match the default name of the collector which I think is `demo`.  Let's change it to `collector`.